src/pages/global.css

```diff
@@ -0,0 +1,12 @@
+html, body, #___gatsby {
+  height: 100%;
+}
+
+body {
+  margin: 0px;
+}
+
+div[role="group"][tabindex] {
```

Comment:
To fix additional wrapper added by gatsby
https://github.com/gatsbyjs/gatsby/issues/7310